Makefile

```diff
@@ -80,11 +80,6 @@ debug: build-debug
 build-release: LDFLAGS += -w
 build-release: pre-build build ## Build a binary without debug information
 
-.PHONY: generate-banzai-cli-docs
-generate-banzai-cli-docs: ## Generate documentation for Banzai CLI
```

Comment:
Right now I'm using this from banzaicloud.com build. My original intent was to keep these generator logics close to the sources, and just call it from the build flow. However it is so simple and probably useless here. So should I add it to the site build instead?

Reply:
no, reverted